### PR TITLE
clean up simple find widget

### PR DESCRIPTION
**Pull request by meganrogge**

### CODE REVIEW

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts [hunk 1]
```diff
@@ -54,15 +54,15 @@ export abstract class SimpleFindWidget extends Widget {
 	private _foundMatch: boolean = false;
 
 	constructor(
-		@IContextViewService private readonly _contextViewService: IContextViewService,
-		@IContextKeyService contextKeyService: IContextKeyService,
-		private readonly _state: FindReplaceState = new FindReplaceState(),
-		private readonly _options: IFindOptions,
+		readonly state: FindReplaceState = new FindReplaceState(),
+		readonly options: IFindOptions,
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:58]: readonly implies public

```suggestion
		options: IFindOptions,
```

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts [hunk 2]
```diff
@@ -54,15 +54,15 @@ export abstract class SimpleFindWidget extends Widget {
...
+		contextViewService: IContextViewService,
+		contextKeyService: IContextKeyService,
 		private readonly _keybindingService?: IKeybindingService
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:61]: I think we should be able to pass this in for `webviewFindWidget` too and make it mandatory

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts [hunk 3]
```diff
@@ -54,15 +54,15 @@ export abstract class SimpleFindWidget extends Widget {
 	private _foundMatch: boolean = false;
 
 	constructor(
-		@IContextViewService private readonly _contextViewService: IContextViewService,
-		@IContextKeyService contextKeyService: IContextKeyService,
-		private readonly _state: FindReplaceState = new FindReplaceState(),
-		private readonly _options: IFindOptions,
-		private readonly _keybindingService?: IKeybindingService
+		readonly state: FindReplaceState = new FindReplaceState(),
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:57]: Oh this was private before too, need to remove readonly?

**Review by Tyriar — APPROVED** [review]

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts (hunk 3 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:57]: yeah I wasn't sure if readonly was used to make it obvious that isn't changed/set

Comment (Tyriar) [reply src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:57]: I've been bitten by doing this exact thing before, don't think it's possible to declare a parameter variable as readonly unless it's a member

Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:57]: options were also readonly before

**Pull request merged**